signal_media_exporter/main.py

```diff
@@ -40,7 +40,12 @@ def get_messages(config, key):
         own_number, device_id = number_id['value'].split('.')
         logger.info('Own number: %s, device ID: %s', own_number, device_id)
 
-        cond = ["hasVisualMediaAttachments > 0"]
+        cond = []
+        include = config.get('includeAttachments', "visual")
+        if include == "visual":
+            cond.append("hasVisualMediaAttachments > 0")
+        elif include == "all":
+            cond.append("hasAttachments > 0")
```

Comment:
This will produce an invalid query error in case the setting is something other than visual or all.
How about raising an explicit exception?